src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1018,7 +1018,9 @@ namespace nupic {
     for (UInt i = 0; i < numColumns; i++) {
       Real perm[8];
       sp.getPermanence(i, perm);
-      NTA_CHECK(check_vector_eq(truePermArr[i], perm, numInputs));
+      print_vec(truePermArr[i], 8);
+      print_vec(perm, 8);
+      NTA_CHECK(check_vector_eq(truePermArr[i], perm, numInputs)) << "Failing: " << truePermArr[i] << " vs. " << perm;
```

Comment:
@scottpurdy what do you think about this change? https://github.com/breznak/nupic.core/commit/ee5bfafaf52c6bdd0c37012c5fcb78c1ba9c23a0
I'd like to have the above in place, but then the test above keeps failing with a small rounding err.